src/test/resources/packagerConfig/simpleConfig.yml

```diff
@@ -9,9 +9,9 @@ casc:
     dir: "casc.yml"
 plugins:
 - groupId: "org.jenkins-ci.plugins"
-  artifactId: "AnchorChain"
+  artifactId: "dotnet-as-script"
```

Comment:
Ok, sort of confused here with this particular plugin.... it looks like it's related to C#?

Reply:
That was just to get the tests pass because the tests had another plugin